acme_diags/plot/cartopy/enso_diags_plot.py

```diff
@@ -0,0 +1,268 @@
+    ax.set_xticks(xticks, crs=ccrs.PlateCarree())
+    ax.set_yticks(yticks, crs=ccrs.PlateCarree())
+    lon_formatter = LongitudeFormatter(
+        zero_direction_label=True, number_format='.0f')
+    lat_formatter = LatitudeFormatter()
+    ax.xaxis.set_major_formatter(lon_formatter)
+    ax.yaxis.set_major_formatter(lat_formatter)
+    ax.tick_params(labelsize=8.0, direction='out', width=1)
+    ax.xaxis.set_ticks_position('bottom')
+    ax.yaxis.set_ticks_position('left')
+    # Place a vertical line in the middle of the plot - i.e. 180 degrees
+    ax.axvline(x=0.5, color='k', linewidth=0.5)
+
+    # Color bar
+    cbax = fig.add_axes(
+        (panel[n][0] + 0.6635, panel[n][1] + 0.0115, 0.0326, 0.1792))
+    cbar = fig.colorbar(contours, cax=cbax)
+    w, h = get_ax_size(fig, cbax)
+
+    if levels is None:
+        cbar.ax.tick_params(labelsize=9.0, length=0)
+
+    else:
+        maxval = np.amax(np.absolute(levels[1:-1]))
+        if maxval < 1.0:
+            fmt = "%5.3f"
+            pad = 30
+        elif maxval < 10.0:
+            fmt = "%5.2f"
+            pad = 25
+        elif maxval < 100.0:
+            fmt = "%5.1f"
+            pad = 25
+        else:
+            fmt = "%6.1f"
+            pad = 30
+        cbar.set_ticks(levels[1:-1])
+        labels = [fmt % l for l in levels[1:-1]]
+        cbar.ax.set_yticklabels(labels, ha='right')
+        cbar.ax.tick_params(labelsize=9.0, pad=pad, length=0)
+
+    # Display stats
+    if stats:
+        top_stats = (stats['max'], stats['min'], stats['mean'], stats['std'])
+        top_text = 'Max\nMin\nMean\nSTD'
+        fig.text(panel[n][0] + 0.6635, panel[n][1] + 0.2107,
+                 top_text, ha='left', fontdict=plotSideTitle)
+        fig.text(panel[n][0] + 0.7635, panel[n][1] + 0.2107,
+                 '%.2f\n%.2f\n%.2f\n%.2f' % top_stats,
+                 ha='right', fontdict=plotSideTitle)
+
+        if 'rmse' in stats.keys():
+            bottom_stats = (stats['rmse'], stats['corr'])
+            bottom_text = 'RMSE\nCORR'
+            fig.text(panel[n][0] + 0.6635, panel[n][1] - 0.0205,
+                     bottom_text, ha='left', fontdict=plotSideTitle)
+            fig.text(panel[n][0] + 0.7635, panel[n][1] - 0.0205, '%.2f\n%.2f' %
+                     bottom_stats, ha='right', fontdict=plotSideTitle)
+
+    # Hatch text
+    if conf is not None:
+        hatch_text = 'Hatched when pvalue < 0.05'
+        fig.text(panel[n][0] + 0.25, panel[n][1] - 0.0355, hatch_text, ha='right', fontdict=plotSideTitle)
+
+
+def plot(reference, test, diff, metrics_dict, ref_confidence_levels, test_confidence_levels, parameter):
+    if parameter.backend not in ['cartopy', 'mpl', 'matplotlib']:
+        return
+
+    # Create figure, projection
+    fig = plt.figure(figsize=parameter.figsize, dpi=parameter.dpi)
+    #proj = ccrs.PlateCarree(central_longitude=180)
+    proj = ccrs.PlateCarree(central_longitude=179.99)
```

Comment:
Maybe we should link to the GitHub issue that forces us to use this central longitude. @chengzhuzhang https://github.com/SciTools/cartopy/issues/946 is the correct one, right?

Reply:
Good idea! Yes it's the correct issue to refer to.